code/modules/clothing/head/armor.dm

```diff
@@ -535,3 +535,18 @@
 	name = "gray tanker helmet"
 	icon_state = "tanker_helmet_gray"
 
+/obj/item/clothing/head/armor/helmet/onestar
+	name = "OneStar officer cap"
+	desc = "A fancy red and blue cap sporting the Onestar insignia."
+	icon_state = "OS_Hat"
```

Comment:
pls avoid using uppercases in icon and item states for the sake of consistency
if you think that `os_hat` or `os_jacket` would not be informative enough you can always change it to `onestar_hat` and `onestar_jacket`

Reply:
changed it